autogen/main/cluster.tf.tmpl

```diff
@@ -577,9 +577,9 @@ resource "google_container_node_pool" "pools" {
     }
     {% if beta_cluster %}
     dynamic "sandbox_config" {
-      for_each = local.cluster_sandbox_enabled
-
+      for_each = tobool((lookup(each.value, "sandbox_enabled", ((local.cluster_sandbox_enabled[0] == "gvisor") ? true : false)))) ? ["gvisor"] : []
```

Comment:
We could use the `sandbox_enabled` variable as the default (as opposed to using `local.cluster_sandbox_enabled[0]`) which would simplify this logic.